SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj

```diff
@@ -183,7 +197,8 @@
 		OBJ_20 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				ENABLE_TESTABILITY = YES;
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
+				ENABLE_BITCODE = NO;
```

Comment:
This seems questionable. We've had problems with bitcode while using Carthage in the past, and I don't want to repeat that. Is it needed?